docs/windows.md

```diff
@@ -201,6 +254,15 @@ Download and apply `antrea-windows.yml`.
 kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-windows.yml
 ```
 
+Since Antrea 1.10, you can also deploy antrea-agent Windows DaemonSet with containerd runtime by applying file `antrea-windows-containerd.yml`.
```

Comment:
```suggestion
Since Antrea 1.10, you can also deploy antrea-agent Windows DaemonSet with Containerd runtime by applying file `antrea-windows-containerd.yml`.
```